bash/formatter.bash

```diff
@@ -9,24 +9,69 @@
 
 function Format_Codebase()
 {
+    local bash_formatter_found='TRUE'
+    local python_formatter_found='TRUE'
+    local bash_format_checked='FALSE'
+    local python_format_checked='FALSE'
+    Ensure_That_Given_Variables_Are_Set_And_Not_Empty HYBRID_top_level_path HYBRID_python_folder
     if ! hash shfmt &> /dev/null; then
-        Print_Fatal_And_Exit \
-            'Command ' --emph 'shfmt' ' not available, unable to format codebase.' \
-            'Please, install it (https://github.com/mvdan/sh#shfmt) and run the formatting again.'
+        Print_Error \
+            'Command ' --emph 'shfmt' ' not available. Please install it (https://github.com/mvdan/sh#shfmt)' \
+            'to format the ' --emph 'Bash' ' files of the codebase.\n'
+        bash_formatter_found='FALSE'
     else
-        Ensure_That_Given_Variables_Are_Set_And_Not_Empty HYBRID_top_level_path
         shfmt -w -ln bash -i 4 -bn -ci -sr -fn "${HYBRID_top_level_path}"
+
```

Comment:
I would fill this spare line with `|| Print_Internal_And_Exit 'Bash formatter ' --emph 'shfmt' ' unexpectedly failed.'` continuing the previous line by `\`. 🤪